validator_client/src/initialized_validators.rs

```diff
@@ -291,6 +319,10 @@ pub struct InitializedValidators {
     validators: HashMap<PublicKeyBytes, InitializedValidator>,
     /// For logging via `slog`.
     log: Logger,
+    /// If true, reuse the same password entered via STDIN for all keystores without password
+    reuse_password: bool,
+    /// Temporarily stores password that will be re-used
+    reuse_password_text: Option<PlainText>,
```

Comment:
I'm not sure if we should have the `PlainText` as a field in the `InitializedValidator` though I realize it's much harder to implement otherwise. I think zeroizing the field after use should be good enough but there might still be potential issues here I'm missing.
cc @paulhauner